calico_containers/pycalico/datastore.py

```diff
@@ -29,6 +29,16 @@
 ETCD_AUTHORITY_DEFAULT = "127.0.0.1:4001"
 ETCD_AUTHORITY_ENV = "ETCD_AUTHORITY"
 
+# Secure etcd with SSL environment variables and paths
+ETCD_SCHEME_DEFAULT = "http"
+ETCD_SCHEME_ENV = "ETCD_SCHEME_ENV"
+ETCD_KEY_FILE_ENV = "ETCD_KEY_FILE"
+ETCD_KEY_NODE_FILE = "/etc/calico/ssl/key.pem"
```

Comment:
These node variables do not belong in libcalico - they are calico-docker specific.